euphrates-0.5.0/README.md

```diff
@@ -0,0 +1,60 @@
+# Babylon Phase 3 Devenet (Euphrates 0.5.0)
+
+The Phase 3 Devenet is a development network of Babylon Genesis Chain. It implements the full funcationality of Bitcoin staking and allow BSNs to be connected. 
+
+## Network Parameters
+
```

Comment:
We can have subsections that

- fee token is `ubbn`
- faucet is at Discord
- versions of the software